tests/lib/rules/space-infix-ops.js

```diff
@@ -24,7 +24,9 @@ eslintTester.addRuleTest("lib/rules/space-infix-ops", {
         "(a + b) + (c + d)",
         "a = b",
         "a ? b : c",
-        "var a = b"
+        "var a = b",
+        { code: "a|0", args: [2, { int32Hint: true }] },
+        { code: "a |0", args: [2, { int32Hint: true }] }
```

Comment:
@nzakas I think what `|0` is a single type cast operator and user can decide how to arrange it. My opinion based on [this message](https://github.com/eslint/eslint/issues/1295#issuecomment-56912360).